jx_sqlite/aggs_table.py

```diff
@@ -122,7 +122,8 @@ def _get(row):
                     push_child=push_child,  # CAN NOT HANDLE TUPLES IN COLUMN
                     pull=pull,
                     sql=sql,
-                    type=sql_type_to_json_type[json_type]
+                    type=sql_type_to_json_type[json_type],
+                    column_alias=None,                                                            
```

Comment:
Can we set the column_alias to something here.

Plus: Be sure to remove that trailing comma!